tap_jira/streams.py

```diff
@@ -340,6 +362,7 @@ def sync(self):
 
 
 VERSIONS = Stream("versions", ["id"], indirect_stream=True)
+BOARDS = BoardsGreenhopper("boardsGreenhopper",["id"])
```

Comment:
Should there be the other stream here as well? When i ran the tap locally, i only see the BoardsGreenhopper table getting created/synced.

Reply:
We, Degreed, are not going to use BoardsAgile in the near future. I could add the BoardsAgile stream here as a comment, but we don't need to run it.